### ISSUE TITLE
[BUG]  browserContext.close: tracing.stopChunk: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by Dennyjomiller98** [microsoft/playwright#10826]
**Context:**
- Playwright Version: [ @playwright/test": "^1.16.3"]
- Operating System: [Linux]
- Node.js version: [14.17.2]
- Browser: [Chromium, Firefox]

## System:
 - OS: Linux 5.4 Ubuntu 20.04.3 LTS (Focal Fossa)
 - Memory: 11.01 GB / 31.32 GB
 - Container: Yes
## Binaries:
 - Node: 14.17.2 - ~/.nvm/versions/node/v14.17.2/bin/node
 - Yarn: 1.9.4 - /usr/bin/yarn
 - npm: 6.14.13 - ~/.nvm/versions/node/v14.17.2/bin/npm
## Languages:
 - Bash: 5.0.17 - /bin/bash
## npmPackages:
 - playwright: ^1.4.2 => 1.4.2 
 - playwright-aws-lambda: ^0.7.0 => 0.7.0 
 - playwright-core: ^1.16.3 => 1.16.3 

**"message": "browserContext.close: tracing.stopChunk: ENOENT: no such file or directory, open '/tmp/playwright-artifacts-89Eepx/2d109266a8902835f00df630f51060a4.trace'"**

This error occurs only on CI (Jenkins pipeline) and is infrequent (only occurs once every 10+ tests).

Reporters:
```javascript
 reporter: [
        ['list'],
        ['json', { outputFile: 'fortifiedResults.json'}]
    ]
```

Projects:
```
projects: [
        {
            name: 'Chromium',
            use: {
                browserName: 'chromium',
                trace: 'retain-on-failure',
                screenshot: 'only-on-failure'
            },
            workers: 2,
        },
        {
            name: 'Firefox',
            use: {
                browserName: 'firefox',
                trace: 'retain-on-failure',
                screenshot: 'only-on-failure'
            },
            workers: 2,
        }
    ]
```

The tests cover both Firefox and Chromium browsers, but it is always only one test that fails.

**Comment by dgozman** [comment]
I think this was fixed in v1.17.1. Could you please upgrade and check for yourself?

**Comment by mxschmitt** [comment]
Closing since this is most likely already fixed. Please create a new issue if you still encounter unexpected behaviour.
